### DISCUSSION TITLE
[DISCUSSION] Looking for chat platforms

### DISCUSSION THREAD

**Discussion by thealidev** [iBotPeaches/Apktool#3337] [Q&A] [answered]
Is there is any community chat platforms of apktool
I prefer any telegram channels or discord servers
If yes then send me link here
Thanks

**Comment by iBotPeaches** [comment, marked as answer]
There is not. I don't find those mediums to be conductive of what Apktool requires, which is heavy amounts of logs and large files.

You've already found the IRC channel, but you left within minutes. Give it a day or two of waiting and you'll get a message in there.

Otherwise - the preferred location is the actual discussion tab or issues for bugs. In this case you've posted a discussion as an issue when there is an actual discussion board - so I'll convert this for you and close it.
